### DISCUSSION TITLE
Is Coolify designed to only be useful for very simple applications?

### DISCUSSION THREAD

**Discussion by dnk8n** [coollabsio/coolify#818] [Q&A] [answered]
I feel like there are not enough features to deploy something like this: https://github.com/dnk8n/saleor-platform/blob/main/docker-compose.yml

Or am I mistaken? Can Coolify handle this?

**Comment by peaklabs-dev** [comment, marked as answer]
This works with v4 via the GitHub type and the docker compose deployment type.
